w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -31,7 +31,8 @@ void dynamicTests(String type, int[] expected, int[] input) {
 
 		int j = 0;
 		for (int i : tree) {
```

Comment:
Und das i hier `element`.

Reply:
Du meinst position und Element in der Err Nachricht mitteilen? Das verstehe ich nicht ganz.

Reply:
Ah stopp die Variablen. Mach ich gleich!